lean4-info.el

```diff
@@ -82,19 +82,26 @@
 (lsp-defun lean4-diagnostic-full-end-line ((&lean:Diagnostic :full-range (&Range :end (&Position :line))))
   line)
 
-(defun lean4-mk-message-section (caption errors)
+(defun lean4-mk-message-section (uri caption errors)
   (when errors
     (magit-insert-section (magit-section)
       (magit-insert-heading caption)
       (magit-insert-section-body
         (dolist (e errors)
           (-let (((&Diagnostic :message :range (&Range :start (&Position :line :character))) e))
-            (magit-insert-section (magit-section)
-              (magit-insert-heading (format "%d:%d: " (1+ (lsp-translate-line line)) (lsp-translate-column character)))
+	    (magit-insert-section (magit-section)
+	      (magit-insert-heading)
+	      (insert-button (format "%s %d:%d" uri (1+ (lsp-translate-line line)) (lsp-translate-column character))
```

Comment:
This prints out the raw URI, which is like:

```
file:///home/bollu/work/oss/lean4-metaprogramming-book/lean/extra/attributes.lean 56:0
```

I imagine there is a way to get the path relative to the LSP root, which I don't know yet.

Reply:
The info view will only ever show information from a single file anyway, no? So why include it at all?